packages/flutter_goldens/lib/src/flutter_goldens_io.dart

```diff
@@ -3,14 +3,16 @@
 // found in the LICENSE file.
 
 import 'dart:async' show FutureOr;
-import 'dart:io' as io show OSError, SocketException;
+import 'dart:convert' show jsonEncode;
+import 'dart:io' as io show File, OSError, SocketException, stdout;
 
 import 'package:file/file.dart';
 import 'package:file/local.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter_goldens_client/skia_client.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:platform/platform.dart';
+import 'package:test_api/src/expect/async_matcher.dart' show AsyncMatcher; // ignore: implementation_imports
```

Comment:
Would we be able to avoid this ignore if `matchesGoldenFile` was overridden instead?

Reply:
`matchesGoldenFile` is a top-level function. I can't think of a way to override it. Any ideas?

Reply:
You could use the same name and hide it from the namespace? Not sure. If there is a way to not expose this and have this ignore that would be ideal.

Reply:
It's exposed from `package:flutter_test/flutter_test.dart`. According to `git grep` there are 874 imports of `flutter_test.dart`. Does the benefit of hiding it outweigh the cost of the noise across the codebase?

Reply:
I dunno! Just brainstorming. I think in general there are a lot of options here. :)